src/esp/assets/FRLInstanceMeshData.h

```diff
@@ -23,20 +20,17 @@ namespace assets {
  * id_to_label and id_to_node map face id to instance and node ids
  * Faces are assumed to be quads
  */
-class InstanceMeshData : public BaseMesh {
+class FRLInstanceMeshData : public GenericInstanceMeshData {
  public:
-  struct RenderingBuffer {
-    Magnum::GL::Mesh mesh;
-    Magnum::GL::Buffer vbo;
-    Magnum::GL::Buffer cbo;
-    Magnum::GL::Buffer ibo;
-  };
-
-  InstanceMeshData() : BaseMesh(SupportedMeshType::INSTANCE_MESH){};
-  virtual ~InstanceMeshData(){};
+  FRLInstanceMeshData()
+      : GenericInstanceMeshData(SupportedMeshType::INSTANCE_MESH){};
+  virtual ~FRLInstanceMeshData(){};
 
   bool from_ply(const std::string& ply_file);
```

Comment:
Can we just rename from_ply to loadPLY, and make it override base class? I think from_ply is only called from couple places.

Reply:
Done